plugins/GSdx/GSSetting.cpp

```diff
@@ -121,6 +121,11 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Enables external shader for additional post-processing effects.";
 		case IDC_FXAA:
 			return "Enables fast approximate anti-aliasing. Small performance impact.";
+		case IDC_AUTO_FLUSH:
+			return "Force a primitive flush when framebuffer is also an input texture. Warning: it is very costly on the performances. Help to fix some processing effects such as Jak shadows, GTA radiosity..."
+				"Note: openGL HW renderer is able to handle Jak shadows at full speed without this option.";
+		case IDC_UNSCALE_POINT_LINE:
+			return "Line width increase from 0 to 1 pixel. Scaling will keep line proportion and it will avoid aliasing. Avoid empty line in the screen for lines based rendered(Ridge Racer V).";
```

Comment:
How about this:
`Increases the width of lines at higher than native resolutions, to ensure that the lines will keep the correct proportions and prevent aliasing. Avoids empty lines on the screen in games such as Ridge Racer V.`